PraceDomowe/PracaDomowa5/Łoń Zofia/PD_5.ipynb

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
<p>OK, chociaż uwzględniasz tylko te pytania które mają jakąś zaakceptowaną odpowiedź. Liczba wszystkich pytań to liczba wierszy w Posts dla których PostTypeId == 1.</p><p> </p><p><code>2 punkty</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>